test/e2e/webhook_test.go

```diff
@@ -424,3 +426,1119 @@ var _ = Describe("webhook tests for MC taints", Ordered, func() {
 		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
 	})
 })
+
+var _ = Describe("webhook tests for ClusterResourceOverride CREATE operations", func() {
+	croName := fmt.Sprintf(croNameTemplate, GinkgoParallelProcess())
+	selector := placementv1beta1.ClusterResourceSelector{
+		Group:   "rbac.authorization.k8s.io/v1",
+		Kind:    "ClusterRole",
+		Version: "v1",
+		Name:    fmt.Sprintf("test-clusterrole-%d", GinkgoParallelProcess()),
+	}
+	policy := &placementv1alpha1.OverridePolicy{
+		OverrideRules: []placementv1alpha1.OverrideRule{
+			{
+				ClusterSelector: &placementv1beta1.ClusterSelector{
+					ClusterSelectorTerms: []placementv1beta1.ClusterSelectorTerm{
+						{
+							LabelSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"key": "value",
+								},
+							},
+						},
+						{
+							LabelSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"key1": "value1",
+								},
+							},
+						},
+					},
+				},
+				JSONPatchOverrides: []placementv1alpha1.JSONPatchOverride{
+					{
+						Operator: placementv1alpha1.JSONPatchOverrideOpRemove,
+						Path:     "/meta/labels/test-key",
+					},
+				},
+			},
+		},
+	}
+
+	It("should deny create CRO with label selector", func() {
+		Eventually(func(g Gomega) error {
```

Comment:
why is this eventually? shouldn't this fail consistently?

Reply:
most of them should be UT

Reply:
Changed create test cases to consistently, but update fails with consistently because object is being modified.